docs/using/syntax-optional.md

```diff
@@ -42,21 +212,31 @@ This text is **standard** _Markdown_
 :::
 ::::
 
-The supported directives are: admonition, attention, caution, danger, error, important, hint, note, seealso, tip and warning.
-
-These directives do **not** currently allow for parameters to be set, but you can add additional CSS classes to the admonition as comma-delimited arguments after the directive name. Also `admonition` can have a custom title.
-For example:
+and also parameter options can be used:
 
 ```md
-:::{admonition,warning} This *is* also **Markdown**
+:::{admonition} This *is* also **Markdown**
+:class: warning
+
 This text is **standard** _Markdown_
 :::
 ```
 
-:::{admonition,warning} This *is* also **Markdown**
+:::{admonition} This *is* also **Markdown**
+:class: warning
+
 This text is **standard** _Markdown_
 :::
 
+(syntax/admonitions)=
+
+## Admonition directives
+
+:::{important}
+`myst_admonition_enable` is deprecated and replaced by `myst_enable_extensions = ["colon_fence"]` (see above).
```

Comment:
just a note that I think adding a list of extensions to enable is a much better pattern than having a unique variable for each 👍

One extra thought I had - what if we used a single configuration variable like `myst_config` and then had keys / values for our various configurations, rather than using a different variable for each that started with `myst_`?

Reply:
meh, I don't think that is necessary or wanted: I believe it would make it difficult in jupyter-book for people to override the defaults of specific aspects of myst-parser, because they would then have to specify the whole dictionary

Reply:
makes sense, I think there are pros/cons just from a "readability" perspective

e.g. I think that for a medium-large number of config over-rides:

```
myst:
  extensions:
    - a
    - b
  otheroption: otheykey
```

is more readable than

```
myst_extensions: a
myst_otheroption: b
```

but anyway discussing this I think should be left to the future and need not block this PR